dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphUtility/BarBrushHelper.cs

```diff
@@ -0,0 +1,122 @@
+﻿//  ---------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+// 
+//  The MIT License (MIT)
+// 
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+// 
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+// 
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//  ---------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Windows.UI;
+using Windows.UI.Composition;
+
+namespace BarGraphUtility
+{
+    // Create brushes to fill the bars. 
+    sealed class BarBrushHelper
+    {
+        private readonly Compositor compositor;
+        private Random rand;
+
+        public BarBrushHelper(Compositor c)
+        {
+            compositor = c;
+            rand = new Random();
+        }
+
+        internal CompositionBrush GenerateSingleColorBrush(Color color)
+        {
+            return compositor.CreateColorBrush(color);
+        }
+
+
+        internal CompositionBrush[] GenerateRandomColorBrushes(int numBrushes)
+        {
+            var brushes = new CompositionBrush[numBrushes];
+            for (int i = 0; i < numBrushes; i++)
+            {
+                byte[] rgb = new byte[3];
```

Comment:
Tip for generating random colors: random rgb values are not a good choice for generating pretty colors as they will happily generate lots of boring low-saturation colors (i.e. grays). Typically you want highly saturated bright colors, which is best done by generating in HSL and just making the H value random.